library/std/src/os/windows/process.rs

```diff
@@ -181,6 +181,13 @@ pub trait CommandExt: Sealed {
     #[stable(feature = "windows_process_extensions", since = "1.16.0")]
     fn creation_flags(&mut self, flags: u32) -> &mut process::Command;
 
+    /// Sets the field [wShowWindow][1] of [STARTUPINFO][2] that is passed to `CreateProcess`.
+    ///
+    /// [1]: https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-showwindow
+    /// [2]: https://learn.microsoft.com/es-es/windows/win32/api/processthreadsapi/ns-processthreadsapi-startupinfow
+    #[unstable(feature = "windows_process_extensions_show_window", issue = "none")]
```

Comment:
Please open a new tracking issue (<https://github.com/rust-lang/rust/issues/new?template=library_tracking_issue.md&title=Tracking+issue+for+CommandExt%3A%3Ashow_window>) and add the issue number into this attribute.

Reply:
Done! Find the tracking issue at #127544